src/components/common/preLoader/PreLoader.tsx

```diff
@@ -0,0 +1,5 @@
+import Style from './PreLoader.module.scss'
+
+export default function PreLoader() {
+  return <span className={Style['loader']} />
```

Comment:
any reason why do we have a loader? since the current implementation is using rsc?